backend/src/test/java/com/woowacourse/ternoko/service/InterviewServiceTest.java

```diff
@@ -135,10 +138,24 @@ void createPreviousDay() {
         final Long interviewId = interviewService.create(허수달.getId(), 면담생성요청정보_준_2022_07_01_10_00);
 
         // then
-        final InterviewResponse interviewResponse = interviewService.findInterviewResponseById(interviewId);
+        final InterviewResponse interviewResponse = interviewService.findInterviewResponseById(허수달.getId(),
+                interviewId);
         assertThat(interviewResponse.getId()).isNotNull();
     }
 
+    @Test
+    @DisplayName("크루 - 본인 소유가 아닌 면담을 조회 시 예외가 발생한다.")
+    void findInterview_WhenFindNotOwnInterview() {
+        // given
+        coachService.putAvailableDateTimesByCoachId(준.getId(), 면담가능시간생성요청정보_2022_07_01_10_TO_12);
+        현재시간_설정(2022, 6, 20, 10, 0);
+        final Long interviewId = interviewService.create(허수달.getId(), 면담생성요청정보_준_2022_07_01_10_00);
+
+        // when
+        assertThatThrownBy(() -> interviewService.findInterviewResponseById(김록바.getId(), interviewId))
+                .isInstanceOf(InvalidInterviewMemberException.class);
```

Comment:
r : 현재시간은 beforeEach에서 설정해주고 있어서 안해도 될거같아요.
위의 테스트코드는 전날예약이라서 시간을 이동해주고 있었습니다~

```suggestion
    void findInterview_WhenFindNotOwnInterview() {
        // given
        coachService.putAvailableDateTimesByCoachId(준.getId(), 면담가능시간생성요청정보_2022_07_01_10_TO_12);

        // when
        final Long interviewId = interviewService.create(허수달.getId(), 면담생성요청정보_준_2022_07_01_10_00);

        // then
        assertThatThrownBy(() -> interviewService.findInterviewResponseById(김록바.getId(), interviewId))
                .isInstanceOf(InvalidInterviewMemberException.class);
```